_data/other.yml

```
@@ -116,6 +116,16 @@ websites:
     tfa:
       - totp
 
+  - name: Clover
+    url: https://www.clover.com
+    img: clover.svg
+    tfa:
+      - totp
+      - sms
+    doc: https://www.clover.com/help/two-factor-authentication
+    regions:
+      - us
```

```suggestion
      - us
      - ca
      - gb
      - ie
      - de
      - at
      - nl
      - ar
```
The site is available in more regions than just the US - e.g. https://www.la.clover.com/ar/ayuda/autenticaci%C3%B3n-de-dos-factores/

---

Done ✅